File: for.cpp
#include <iostream>
using namespace std;

int main()
{
    int bil;
  
    for (int i=1; i<=3; i++){
    
    cout<< "Masukkan Bilangan : "<<endl;
    cin>>bil;
    cout<<endl;
    
    if (bil%2==0)
    {
        cout<<"merupakan bilangan genap"<<endl;
    }
    else
    {
        cout<<"merupakan bilangan ganjil"<<endl;
    }
    }

    return 0;
}

File: for.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner myObj = new Scanner(System.in);
        String y;
        int x;
        
        for (int i=1; i <= 6; i++){
        System.out.println("Masukkan bilangan: ");
        y = myObj.nextLine();
        x = Integer.parseInt(y);
        
        if (x%2==0)
            System.out.println("Bilangan " + x + " merupakan bilangan genap\n");
        else
            System.out.println("Bilangan " + x + " merupakan bilangan ganjil\n");
            
        }
    }
}